pkg/ingress/model_build_listener_rules.go

```diff
@@ -39,9 +39,12 @@ func (t *defaultModelBuildTask) buildListenerRules(ctx context.Context, lsARN co
 				if err != nil {
 					return errors.Wrapf(err, "ingress: %v", k8s.NamespacedName(ing.Ing))
 				}
-				tags, err := t.buildListenerRuleTags(ctx, ing)
-				if err != nil {
-					return errors.Wrapf(err, "ingress: %v", k8s.NamespacedName(ing.Ing))
+				tags := map[string]string{}
+				if t.featureGate.Enabled(EnableListenerRulesTagging) {
```

Comment:
actually, let's minimize the changes, we don't need to change the model_builder sections, and let it continue build tags as normal.

These tags built here won't be used in model_deployer anyway when EnableListenerRulesTagging is not enabled.